app/models/publisher.rb

```diff
@@ -159,6 +164,36 @@ def owner_identifier
     "publishers#uuid:#{id}"
   end
 
+  def promo_status(promo_running)
```

Comment:
i think we should put model promo methods in a model Concern (this one would be a Concern just for Publisher). for cleanliness

https://stackoverflow.com/questions/14541823/how-to-use-concerns-in-rails-4